sass/components/_search.scss

```diff
@@ -181,6 +186,65 @@
         right: 25px;
         font-size: 1em;
     }
+
+    // global header style adjustments
+    &.search__form--global {
+        background: var(--sidebar-text-08);
```

Comment:
This could be left for a followup ticket since I assume they're just moving from elsewhere, but these should be using rgba to set opacity